<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>518</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>473</width>
    <height>518</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>473</width>
    <height>518</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Обработчик результатов тестов</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="authInfoBox">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Данные для входа</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLabel" name="GauthIco">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>ico/question.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="VKauthLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Файл аутентификации VK не найден</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="GauthLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Файл аутентификации Google не выбран</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="VKauthIco">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="pixmap">
          <pixmap>ico/cross.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="VKauthBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="GauthBtn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>82</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>82</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="functionBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Работа с результатами тестов</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="groupLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Целевое сообщество — ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="adminLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Выбранный администратор — ?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="msgsNumLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Всего сообщений в беседе — ?</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="tipLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Введите вопрос из теста, результаты которого нужно обработать...
Например:
&quot;У него в сумке лежали учебники для сегодняшних уроков, обед и...&quot;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="targetQLine">
             <property name="font">
              <font>
               <family>Noto Sans</family>
               <pointsize>11</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>133</width>
               <height>38</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>133</width>
               <height>38</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Собрать ответы</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>445</width>
           <height>154</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>445</width>
           <height>154</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="consoleOut">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Прогресс выполнения...&lt;/span&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="outputBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="txtCheck">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Вывод как *.txt</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="csvCheck">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Вывод в *.csv</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="googleCheck">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Вывод в Google Drive</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="saveBtn">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Вывод ответов</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>VKauthBtn</tabstop>
  <tabstop>GauthBtn</tabstop>
  <tabstop>targetQLine</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>consoleOut</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>VKauthBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>VKauthBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GauthBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>GauthBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>StartBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>targetQLine</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnsDisable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>csvCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnsDisable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnsDisable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>googleCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>BtnsDisable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SaveBtn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>562</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>VKauthBtn_clicked()</slot>
  <slot>GauthBtn_clicked()</slot>
  <slot>StartBtn_clicked()</slot>
  <slot>BtnsDisable()</slot>
  <slot>SaveBtn_clicked()</slot>
 </slots>
</ui>
